test/test.js

```diff
@@ -33,6 +33,13 @@ describe('chai-immutable', function () {
     });
 
     describe('equal method', function () {
+      it(
+        'fails when only the "expected" value is an Immutable collection',
+        function () {
+          assert.throws(expect([]).to.equal.bind(null, List()));
```

Comment:
`bind` is just used to create a partially applied version of `equal` (instead of wrapping the call manually with a function declaration), but I can update it to declare a new function if you want.

Are you sure this test is passing without my fix? It isn't for me. Can you double check that?